scripts/check_copyright_notice.py

```diff
@@ -38,7 +38,7 @@
 )?# -\*- coding: utf-8 -\*-
 # ------------------------------------------------------------------------------
 #
-#   (Copyright 2018-(2019|2020) Fetch.AI Limited|Copyright [0-9]{4}(-[0-9]{4})? [a-zA-Z_]+)
+#   (Copyright 2018-(2019|2020|2021) Fetch.AI Limited|Copyright [0-9]{4}(-[0-9]{4})? [a-zA-Z_]+)
```

Comment:
probably automatic substitution with list of valid years?

Reply:
good idea

Reply:
@DavidMinarsch although I'm thinking that maybe we should update everything to `2018-2021`. Thoughts?

Reply:
No! That's not a good idea. The diff will be absolutely massive and there is no gain from it.

Reply:
apart from the diff. I am wondering about the "right" thing to do. The copyright should be extended for early Python modules/files. It clearly has no practical gain, as often when dealing with such "legal" things.

Reply:
No the copyright does not need to be extended for the files we don't touch. We discussed this last year and came to the conclusion that this is ok.

Reply:
I think in 2020 we have touched some of those added in 2019, but we haven't done that. Anyway, not very important as I said.